src/components/Toast.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from "react";
+import "./Toast.scss";
+
+export const Toast = ({
+  message,
+  clearToast,
+}: {
+  message: string | null;
+  clearToast: Function;
+}) => {
+  const [shouldClear, setShouldClear] = useState(true);
+
+  useEffect(() => {
+    if (message !== null) {
+      const timeout = setTimeout(() => {
+        if (shouldClear) {
+          clearToast();
+        }
+      }, 2000);
```

Comment:
```suggestion
      }, TOAST_TIMEOUT);
```

and add it in `constatnts.ts`